test/libsolidity/SolidityExpressionCompiler.cpp

```diff
@@ -162,6 +162,12 @@ bytes compileFirstExpression(
 				context << context.functionEntryLabel(dynamic_cast<FunctionDefinition const&>(
 					resolveDeclaration(*sourceUnit, function, resolver)
 				));
+
+			context.appendMissingLowLevelFunctions();
+			// NOTE: We intentially disable optimisations for utility functions to simplfy the tests
+			context.appendYulUtilityFunctions({});
```

Comment:
To avoid testing complexity let just disable the optimiser for utility functions. We should perhaps remove this entire test suite.